### ISSUE TITLE
No src->dest paths in copy_file()

### ISSUE THREAD

**Issue by annwhoorma** [annwhoorma/DS20-project-2#3]
https://github.com/annwhoorma/DS20-project-2/blob/eb981bb82c527c45d4246012e42ad56d0a84342b/server.py#L108

For example, in the function ```move_file()``` there are ```src``` and ```dest``` variables, but not in ```copy_file()``` but they should be here as well, right?

**Comment by screemix** [comment]
No, the copying command could copy one in the current directory.
To perform actions you wrote here, just do -> copy -> move
